packages/flame/lib/src/collisions/hitboxes/shape_hitbox.dart

```diff
@@ -83,6 +94,14 @@ mixin ShapeHitbox on ShapeComponent implements Hitbox<ShapeHitbox> {
       _parentSizeListener?.call();
       hitboxParent.size.addListener(_parentSizeListener!);
     }
+
+    // This should be placed after the hitbox parent listener
+    // since the correct hitbox size is required by the QuadTree.
+    final parentGame = findParent<FlameGame>();
```

Comment:
Use `findGame` instead

Reply:
This is not my piece of code, I just changed it's place ;-)
![image](https://user-images.githubusercontent.com/1258285/189950460-c3a4fce7-1827-4edf-ba07-06b515158b67.png)